### PR TITLE
[dagster-airlift] mirroring to tutorial example

### PR DESCRIPTION
**Pull request by dpeng817**
Automated tooling to push up tutorial example to mirrored public repo.
One main new script mirror_tutorial.sh, and moving some stuff around + changes to the readme.

Tested this by running through it live and making sure it correctly updates the tutorial repo: https://github.com/dagster-io/airlift-tutorial
Then, running back through the release process to ensure that all scripts I've changed work.

### CODE REVIEW

**Comment by dpeng817** [comment]
* **#24080** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24080?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @dpeng817 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: examples/experimental/dagster-airlift/examples/tutorial-example/README.md [hunk 1]
```diff
@@ -2,11 +2,10 @@
...
 This tutorial will walk through the process of peering, observing, and migrating assets from an Airflow DAG to Dagster.
 
-For now, you have to check out dagster's monorepo to get this code, but we will change that soon - thanks for your patience.
-
+First, clone the tutorial example repo locally, and enter the repo directory.
 ```bash
-gh repo clone dagster-io/dagster
-pushd dagster/examples/experimental/dagster-airlift/examples/tutorial-example
+gh repo clone dagster-io/airlift-tutorial
+pushd airlift-tutorial
```
Comment (schrockn) [inline examples/experimental/dagster-airlift/examples/tutorial-example/README.md:8]: probably can just `cd` here

File: examples/experimental/dagster-airlift/examples/tutorial-example/README.md [hunk 2]
```diff
@@ -2,11 +2,10 @@
 
 This tutorial will walk through the process of peering, observing, and migrating assets from an Airflow DAG to Dagster.
 
-For now, you have to check out dagster's monorepo to get this code, but we will change that soon - thanks for your patience.
-
+First, clone the tutorial example repo locally, and enter the repo directory.
 ```bash
-gh repo clone dagster-io/dagster
-pushd dagster/examples/experimental/dagster-airlift/examples/tutorial-example
+gh repo clone dagster-io/airlift-tutorial
```
Comment (schrockn) [inline examples/experimental/dagster-airlift/examples/tutorial-example/README.md:7]: ugh so much better

File: examples/experimental/dagster-airlift/scripts/mirror_tutorial.sh [hunk 3]
```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```
Comment (schrockn) [inline examples/experimental/dagster-airlift/scripts/mirror_tutorial.sh:1]: If these are copy pasted from elsewhere can you indicate from where so we can consolidate at some point?

**Review by schrockn — APPROVED** [review]
ok great

File: examples/experimental/dagster-airlift/scripts/mirror_tutorial.sh (hunk 3 above)
Comment (dpeng817) [reply examples/experimental/dagster-airlift/scripts/mirror_tutorial.sh:1]: Not quite copy pasted, but yea I think some consolidation is in order

Comment (schrockn) [reply examples/experimental/dagster-airlift/scripts/mirror_tutorial.sh:1]: "Inspired by"

**Review by schrockn — APPROVED** [review]

File: examples/experimental/dagster-airlift/scripts/mirror_tutorial.sh (hunk 3 above)
Comment (dpeng817) [reply examples/experimental/dagster-airlift/scripts/mirror_tutorial.sh:1]: 😜

**Pull request merged**